packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.test.tsx

```diff
@@ -199,9 +200,9 @@ await testIn("browser")(
     await findByText("User");
 
     assert.ok(attemptedRenderCount > 0);
-    // one render to rehydrate the server value
+    // will try with server value and immediately restart with client value
     // one rerender with the actual client value (which is hopefull equal)
-    assert.equal(finishedRenderCount, 2);
+    assert.equal(finishedRenderCount, 1);
```

Comment:
This is a nice improvement since it removes a double render that would in reality only be necessary on a server<->client race condition.